tests/test_dss.py

```diff
@@ -613,12 +614,15 @@ def test_getters(self, mcd):
         assert isinstance(mcd.pot.rho(), datetime)
 
         assert mcd.pot.pie() >= Wad(0)
-        assert mcd.pot.dsr() > Ray(0)
+        assert mcd.pot.dsr() >= Ray.from_number(1)
         assert datetime.fromtimestamp(0) < mcd.pot.rho() < datetime.utcnow()
 
     def test_drip(self, mcd):
+        chi_before = mcd.pot.chi()
+        assert isinstance(chi_before, Ray)
         assert mcd.pot.drip().transact()
-
+        chi_after = mcd.pot.chi()
+        assert chi_before < chi_after
```

Comment:
If the DSR is set to 1 RAY (i.e. 0%) when `mcd.pot.drip().transact()` is called, then `chi_before == chi_after`. Therefore, I suggest we verify that the DSR is over 1 RAY before comparing chi values.

Reply:
Done.